### DISCUSSION TITLE
How to configure vee-validate4 in nuxt 3 beta

### DISCUSSION THREAD

**Discussion by RosarioDeveloper** [nuxt/nuxt#18258] [Questions]
Hi everybody, I try to install end configure vee-validate4 in nuxt3 beta but receive that error:
SyntaxError: Cannot use import statement outside a module

My script is:

<script lang="ts" >
import { useField } from "vee-validate";

export default {
  layout: "auth",
  setup() {
    function isRequired(value: any) {
      if (value && value.trim()) {
        return true;
      }
      return "This is required";
    }
    const { errorMessage, value } = useField("fieldName", isRequired);
    return {
      errorMessage,
      value,
    };
  },
};
</script>

Someone help please?

**Comment by wobsoriano** [comment]
If adding to `build.transpile` didn't make it work, check this issue https://github.com/logaretm/vee-validate/issues/3550

**Comment by jankaderabek** [comment]
See https://github.com/nuxt/framework/issues/1235#issuecomment-951151498
